src/containers/Layout/Sidebar.js

```diff
@@ -432,8 +432,7 @@ function Sidebar({ history, settings, setSetting, getGovernanceVenus }) {
             />
           </svg>
           <div className="flex flex-column align-center">
-            <Label primary>Transaction</Label>
-            <Label primary>History</Label>
```

Comment:
Lol

Reply:
cost me some time to figure out how to come up with this kind of code lol